xml/System.Threading/Mutex.xml

```diff
@@ -67,7 +67,10 @@
 > [!NOTE]
 >  On a server that is running Terminal Services, a named system mutex can have two levels of visibility. If its name begins with the prefix "Global\\", the mutex is visible in all terminal server sessions. If its name begins with the prefix "Local\\", the mutex is visible only in the terminal server session where it was created. In that case, a separate mutex with the same name can exist in each of the other terminal server sessions on the server. If you do not specify a prefix when you create a named mutex, it takes the prefix "Local\\". Within a terminal server session, two mutexes whose names differ only by their prefixes are separate mutexes, and both are visible to all processes in the terminal server session. That is, the prefix names "Global\\" and "Local\\" describe the scope of the mutex name relative to terminal server sessions, not relative to processes.  
   
-   
+> [!NOTE]
+> Backslash (\\) is a reserved character in Mutex name. Do not use backslash (\\) in mutext name except as specified above otherwise <xref:System.IO.DirectoryNotFoundException> may be thrown (even though the name of the mutext represents existing file).
```

Comment:
Backslash --> The backslash
in Mutex name --> in a mutex name
use backslash --> use a backslash
in mutext name --> in a mutex name
as specified above --> as specified in the note on using mutexes in terminal server sessions.
otherwise --> Otherwise, a 
mutext --> mutex

These changes should also be made to each of the remaining three notes on the backlash character.